{{cookiecutter.repostory_name}}/.github/dependabot.yml

```diff
@@ -4,7 +4,9 @@ updates:
     directory: "/app/src"
     schedule:
       interval: "daily"
+    open-pull-requests-limit: 0
```

Comment:
https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#overriding-the-default-behavior-with-a-configuration-file